crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -113,7 +113,7 @@ def compute_status
           )
         end
         nova = NodeObject.find("roles:nova-controller").first
-        ret[:no_live_migration] = true unless nova["nova"]["use_migration"]
```

Comment:
But with this PR the `no_live_migration` will be reported if there's no nova. If I understand correctly this is not the right logic.